MiniSim/MenuItems/DeviceListSection.swift

```diff
@@ -8,15 +8,21 @@
 import Foundation
 
 enum DeviceListSection: Int, CaseIterable {
-    case iOS = 100
-    case android
+    case iOSPhysical = 100
+    case iOSVirtual
+    case androidPhysical
+    case androidVirtual
 
     var title: String {
         switch self {
-        case .iOS:
+        case .iOSPhysical:
+            return NSLocalizedString("iOS Devices", comment: "")
+        case .iOSVirtual:
             return NSLocalizedString("iOS Simulator", comment: "")
```

Comment:
Let's make every menu section in the plural form "iOS Simulators", "Android Simulators"

Reply:
Sounds like a good idea. What about also changing `Android Simulator` to `Android Emulators`? You also use `emulator` in raycast

Reply:
Do you think it is better to also change `Android Simulators` to `Android Emulators`?

Reply:
Oh sure, let's do this!

Reply:
done